ArduPlane/Parameters.cpp

```diff
@@ -1543,6 +1543,17 @@ void Plane::load_parameters(void)
 #if AP_FENCE_ENABLED
     AP_Param::convert_class(g.k_param_fence, &fence, fence.var_info, 0, 0, true);
 #endif
+  
+    // PARAMETER_CONVERSION - Added: Dec 2023
+    // Convert _CM (centimeter) parameters to meters and _CD (centidegrees) parameters to meters
+    {
+        const AP_Param::ConversionInfo centimeters_conversion_info[] {
+            { Parameters::k_param_pitch_trim_cd, 0, AP_PARAM_FLOAT, "PITCH_TRIM" },
+        };
+        AP_Param::convert_old_parameters(&centimeters_conversion_info[0], ARRAY_SIZE(centimeters_conversion_info), 0.02f, 0);
+    }
+    // Create a shim to enable the existing code to work, which should be fixed at a later date
+    g.pitch_trim_cd.set((int16_t)(g.pitch_trim * 100.0f));
```

Comment:
Can't do this, it would mean you would have to reboot to get the param to work.

Reply:
In the dev call on Monday I remember Tridge suggesting this I think:- just changing the parameter name without changing the code. Do you think he missed that this won't work or has some "Tridge" magic idea to make it happen?

Reply:
The code change in that case was related to the position controller which uses centimeters internally as well as via the exposed params. This method is fine, it just can't be a one time thing at boot. You could add a function `get_pitch_trim_cm` with does the `pitch_trim * 100.0` then just use it in place of the old param where ever it is used.

Reply:
I might as well just fix it properly then.

Reply:
I removed the shim and fixed all the usages of pitch_trim_cd - there weren't too many.